File: easy/pallindrome-number.ts
function isPalindrome(x: number): boolean {
    if (x < 0) {
        return false
    }

    let input = x
    let reverseDigit = 0
    while (x > 0) {
        let lastDigit = x%10
        reverseDigit = reverseDigit*10 + lastDigit
        x = Math.floor(x/10)
    }
    return input == reverseDigit
};

File: easy/pallindrome-number.go
package easy

func isPalindrome(x int) bool {
	if x < 0 {
		return false
	}

	input := x
	reversedigit := 0

	for x > 0 {
		lastdigit := x % 10
		reversedigit = reversedigit*10 + lastdigit

		x = x / 10
	}
	return input == reversedigit
}